src/Browscap/Data/PropertyHolder.php

```diff
@@ -136,10 +136,11 @@ public function getPropertyType($propertyName)
      * Determine if the specified property is an property that should
      * be included in the "full" versions of the files only
      *
-     * @param  string $propertyName
+     * @param  string                           $propertyName
+     * @param  \Browscap\Writer\WriterInterface $writer
      * @return bool
      */
-    public function isLiteModeProperty($propertyName)
+    public function isLiteModeProperty($propertyName, WriterInterface $writer = null)
```

Comment:
Please add type declarations throughout here (parameter and return types please)